### ISSUE TITLE
tuned-gui does not work on Wayland

### ISSUE THREAD

**Issue by michel-slm** [redhat-performance/tuned#118]
On a Fedora 28 laptop, tuned-gtk-2.10.0-4.fc28.noarch:

- when logged in to a GNOME on Xorg session running `tuned-gui` works as expected (I get an authentication prompt and after that the GUI pops up)
- when logged in to a GNOME on Wayland session, `tuned-gui` triggers the authentication prompt, and after that I get this:

```
No protocol specified
Unable to init server: Could not connect: Connection refused
No protocol specified
Unable to init server: Could not connect: Connection refused
```

after which tuned-gui exits with error code 0.

see also Red Hat Bugzilla https://bugzilla.redhat.com/show_bug.cgi?id=1569859

**Comment by michel-slm** [comment]
looks like when run as root:
- `from gi.repository import Gtk` produces the two `Unable to init server` lines
- `builder.add_from_file(GLADEUI` causes a segfault

**Comment by olysonek** [comment]
@TomasKorbar Do you know if this got fixed in https://github.com/redhat-performance/tuned/pull/138?

**Comment by TomasKorbar** [comment]
Yes, this has been fixed in #138

**Comment by olysonek** [comment]
Thanks, closing.
